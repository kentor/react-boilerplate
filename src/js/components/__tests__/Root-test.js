const React = require('react');
const Root = require('../Root');
const { shallow } = require('enzyme');

describe('Root', () => {
  it('renders', () => {
    const wrapper = shallow(<Root />);
    expect(wrapper).toMatchSnapshot();
  });
});
