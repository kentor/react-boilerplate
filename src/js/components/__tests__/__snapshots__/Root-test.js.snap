// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root renders 1`] = `
<main>
  おはようございます。
</main>
`;
